spec/features/resources_load_more_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe "Load more resources", js: true do
+  let(:user) { create :user }
+  feature 'load more feature' do
+    context 'clicking the load more button' do
+      before :each do
+        visit('/resources')
+      end
+
+      it "loads 3 resources" do
+        expect(page).to have_text '3 of 120 resources'
+        expect(page).to have_selector('.Resources__gridThreeItem___1Lm4E', count: 3)
+        expect(page).to have_button('Load more')
+      end
+
+      it 'loads more resources' do
+        2.times do
+          click_button('Load more')
+        end
+
+        expect(page).to have_text '9 of 120 resources'
+        expect(page).to have_selector('.Resources__gridThreeItem___1Lm4E', count: 9)
```

Comment:
This will not pass consistently since it's an autogenerated React component className. We can replace this with a static className we can apply to this element.

Reply:
Yeah. I agree with this. Looking into it 👓

Reply:
raised a PR against this branch addressing this 2 months ago https://github.com/neon98/ifme/pull/2

@julianguyen can I push the updates directly into the branch so that we can ship this PR/Issue?

Reply:
Hi @bmwachajr . I have merged your PR. I am so sorry for the delay :(